src/components/BreedsList.vue

```diff
@@ -1,42 +1,62 @@
 <template>
   <section>
-    <search-form @search="getSearchResults"></search-form>
+    <search-form
+      :isDisabled="isInputDisabled"
+      @search="getSearchResults"
+    ></search-form>
 
-    <div>
+    <div class="result__container">
       <p v-if="isLoading">The Data is Loading ...</p>
-      <p v-else-if="!isLoading && error">{{ error }}</p>
+      <p v-else-if="!isLoading && error" class="result__error">{{ error }}</p>
       <p v-else-if="isTimeOut && !displayedBreeds.length">
         The request takes longer than expected...
       </p>
-      <p v-else-if="!isLoading && !displayedBreeds.length">
+      <p v-else-if="!isLoading && !breedsList.length">
         No Data Found. Please try again later.
       </p>
 
       <ul v-else>
-        <p v-if="isTimeOut && !searchList.length && searchQuery !== ''">
-          The request takes longer than expected...
-        </p>
-        <li
-          v-else-if="
-            !isLoading && searchList.length === 0 && searchQuery !== ''
-          "
-        >
+        <li v-if="!isLoading && searchQuery !== '' && !searchList.length">
           Your searches for "{{ this.searchQuery }}" did not have any matches.
           Try different keywords.
         </li>
 
-        <li v-for="(breed, index) in displayedBreeds" :key="breed.id">
-          <h3>{{ index + 1 }} {{ breed.name }}</h3>
-
-          <span>Temperament: {{ breed.temperament }}</span>
-          <span>Life span: {{ breed.life_span }}</span>
-          <span>Height: {{ breed.height.metric }}cm</span>
-          <span>Weight: {{ breed.weight.metric }}kg</span>
-          <img
-            class="breed_image"
-            :src="`https://cdn2.thedogapi.com/images/${breed.reference_image_id}.jpg`"
-            :alt="`Picture of the ${breed.name} dog`"
-          />
+        <li
+          v-for="(breed, index) in displayedBreeds"
+          :class="isHidden ? 'hidden-result__details' : 'result__list'"
+          :key="breed.id"
+          @click="toggleDetails(index)"
```

Comment:
Ko uporabnik klikne na določeno pasmo, se mu morajo odpreti details glede te pasme, vendar pa mi trenutno odpre vse, čeprav kliknem eno; vem da moram nekako označiti tisto ki sem jo kliknil, zato sem dodal `currentlyActivate `dato, ki je initially null, vendar pa na click event `toggleDetails`, ta `currentlyActive `nastavim na `index `od te pasme. Sedaj pa ne vem, kako naj se lotim, da vsem ostalim spremenim ta currentlyActive to null. Upam da ti je jasno ;)